internal/work/package.go

```diff
@@ -21,10 +21,10 @@ type Package struct {
 
 // ValidateNumberOfEntries (14.4 v0.5.2)
 func (wp *Package) ValidateNumberOfEntries() error {
-	var totalExported, totalImported uint
+	var totalExported, totalImported uint16
```

Comment:
Is this change for efficiency?

Reply:
In the graypaper (GP), `N` is defined as the set of non-negative integers with **no fixed size**. However, I changed `ExportedSegments` to `uint16` to align with the ASN.1 types defined [here](https://github.com/w3f/jamtestvectors/blob/master/jam-types-asn/jam-types.asn#L143).